contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -253,6 +253,7 @@ contract SmartAccount is
         address gasToken,
         address payable refundReceiver
     ) private returns (uint256 payment) {
+        require(tokenGasPriceFactor != 0, "invalid stokenGasPriceFactor");
```

Comment:
is this typo `stokenGasPriceFactor` or am i missing something?

Reply:
typo..fixing it

Reply:
done